plugins/cursecheck.cpp

```diff
@@ -244,29 +193,23 @@ command_result cursecheck (color_ostream &out, vector <string> & parameters)
                 }
 
                 auto death = df::incident::find(unit->counters.death_id);
-                bool missing = false;
-                if (death && !death->flags.bits.discovered)
-                {
-                    missing = true;
-                }
-
                 out.print("born in %d, cursed in %d to be a %s. (%s%s%s)\n",
                     unit->birth_year,
                     unit->curse_year,
-                    curse_names [static_cast<size_t>(cursetype)].c_str(),
+                    curse_names[cursetype].c_str(),
                     // technically most cursed creatures are undead,
                     // therefore output 'active' if they are not completely dead
                     unit->flags2.bits.killed ? "deceased" : "active",
                     unit->flags3.bits.ghostly ? "-ghostly" : "",
```

Comment:
ghostly will be output as part of getReadableName, if you switch to using that above